website/static/js/pages/project-settings-page.js

```diff
@@ -13,28 +13,48 @@ require('css/addonsettings.css');
 
 var ctx = window.contextVars;
 
-
-// Initialize treebeard grid for notifications
 var ProjectNotifications = require('js/notificationsTreebeard.js');
 var $notificationsMsg = $('#configureNotificationsMessage');
 var notificationsURL = ctx.node.urls.api  + 'subscriptions/';
-// Need check because notifications settings don't exist on registration's settings page
-if ($('#grid').length) {
+
+function initNotificationsTB() {
+    // Initialize treebeard grid for notifications
+    // Need check because notifications settings don't exist on registration's settings page
+    if ($('#grid').length) {
+        $.ajax({
+            url: notificationsURL,
+            type: 'GET',
+            dataType: 'json'
+        }).done(function(response) {
+            new ProjectNotifications(response);
+        }).fail(function(xhr, status, error) {
+            $notificationsMsg.addClass('text-danger');
+            $notificationsMsg.text('Could not retrieve notification settings.');
+            Raven.captureMessage('Could not GET notification settings.', {
+                url: notificationsURL, status: status, error: error
+            });
+        });
+    }
+}
+initNotificationsTB();
+
+//Initialize treebeard grid for node admins
+var ProjectMailingList = require('../mailingListTreebeard.js');
+var mailingListSettingsURL = ctx.node.urls.api + 'mailing_list/';
```

Comment:
Pending celery changes are a blocker. They're scheduled for more rigorous QA today, but preliminary evaluation with Celery Flower and RabbitMQ-management indicates that they're good to go. So, it shouldn't be a blocker for long, and the update for that will be as trivial as appending `'website.mailing_list'` and `'scripts.mailing_lists'` to the list of high-priority modules.